### PR TITLE
Products css updates

### PR DESCRIPTION
**Pull request by avc278**
### Assignee Tasks

* [ ] view product cards in brands or departments pages
* [ ] view individual product detail pages

### Guidelines

This fixes CSS on full width views, and adds a new layer of CSS when the user is viewing the page on a phone/tablet

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -70,15 +90,27 @@ form div {
 }
 
 .bestSeller {
-  height: 2.5rem;
+  height: 1.2em;
   width: 40%;
-  font-size: 2rem;
+  font-size: 1em;
```
Comment (robertpengcode) [inline public/style.css:95]: How about just a little bigger on bigger screen? I tested the following. The text won't come out of the banner.
.bestSeller {
  height: 1.5em;
  width: 50%;
  font-size: 1.2em;

File: public/style.css [hunk 2]
```diff
@@ -269,6 +293,17 @@ form div {
...
+  .addToCartBtnDetail {
+    width: 40%;
+    height: 2rem;
+    font-size: 1.2rem;
+    background-color: green;
+    color: white;
+    border-radius: 0.5rem;
+    cursor: pointer;
+  }
+}
```
Comment (robertpengcode) [inline public/style.css:306]: for some reason I don't see the part working... let me know if I'm wrong

File: public/style.css [hunk 3]
```diff
@@ -279,6 +314,17 @@ form div {
...
+  .backToShopBtn {
+    width: 40%;
+    height: 2rem;
+    font-size: 1.2rem;
+    background-color: orange;
+    color: white;
+    border-radius: 0.5rem;
+    cursor: pointer;
+  }
+}
```
Comment (robertpengcode) [inline public/style.css:327]: the same with above. Is it working?

File: public/style.css (hunk 2 above)
Comment (avc278) [reply public/style.css:306]: I just added some functionality in the css to fix what I think you're referring to. I can demo it after stand up before I merge it in

**Pull request merged**